### ISSUE TITLE
Rock Paper Scissors: <Link Variables and Operators: How to run Javascript code.>

### ISSUE THREAD

**Issue by crasion528** [TheOdinProject/curriculum#27457]
### Checks

- [X] This is not a duplicate of an existing issue (please have a look through our [open issues list](https://github.com/TheOdinProject/curriculum/issues) to make sure)
- [X] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
- [X] Would you like to work on this issue?

### Describe your suggestion

My proposed change would be to add a link in the text in `Project: Rock Paper Scissors: Quick exercises before starting: List item 1` to reference `How to run JavaScript code` in the ` Variables and Operators` lesson. For this change to be practical though, a change would also need to be made in `Variables and Operators: How to run JavaScript code`. It would require explaining and showing an example of how to insert JavaScript via inline method. It would be practical for referencing all in one spot later on throughout the curriculum. The example could also explain that it is "not best practice," or "not to worry about this until later,"and any other necessary explanation of the method necessary for the objective of the lesson it is contained in.
Here is the link to the lesson and section that is being mentioned in the latter above: https://www.theodinproject.com/lessons/foundations-variables-and-operators#how-to-run-javascript-code 

### Path

Foundations

### Lesson Url

https://www.theodinproject.com/lessons/foundations-rock-paper-scissors

### (Optional) Discord Name

crasion528

### (Optional) Additional Comments

I would like to work on this but I would need to collaborate with someone who is more experienced because I just got to the `Project: Rock Paper Scissors`. If anybody wouldn't mind collaborating and allowing me to work on what I would be capable of would be appreciated if it was possible.

**Comment by wise-king-sullyman** [comment]
I'm not sure if we need to add the content on inline JS, but I like the idea broadly, thanks for suggesting it! I'll go ahead and assign you.

What do you need help with on it?

**Comment by crasion528** [comment]
Well I'm a little confused about how to incorporate the third way to include JS with a link. If I link `How to run JavaScript code`, there are only two ways. Should we include an extra sentence implying to google for the third way? Or perhaps there is a third way that I missed somewhere in the given material already up to that point. Also, how would you prefer me to link it? Just a link added on at the end, or a piece of the text hyperlinked? @wise-king-sullyman

**Comment by wise-king-sullyman** [comment]
@TheOdinProject/javascript What do yall think about this?

I feel like we don't really need to have dedicated content about inline JS in HTML, but I also don't love sending people off to read about it either.

I'm actually debating if we want to just change that bit in RPS to "two ways".

**Comment by MaoShizhong** [comment]
I'm in favour if changing to "2 ways".

On the one hand, simple awareness of inline JS can make seeing it in the wild less "wth???" but on the other hand, besides beginner material that shows this, where would one likely encounter a serious use of inline JS? AFAIK it's basically "here's how to do it but don't do it".

**Comment by wise-king-sullyman** [comment]
Yeah that's pretty much my thoughts as well. I figure we don't want learners using it in their projects, and if they ever do encounter it "in the wild" they should be equipped to figure it out on their own by then.

**Comment by thatblindgeye** [comment]
I do think that at some point including some mention of inline JS could be fruitful - makes users aware of the different ways to use JS + the general syntax, but also allows us to get ahead of a possible detour/wrong turn from users that eventually find it and think "well why don't I just use this instead then' - especially as a "foundational" path where it would be covered. But that's not something I think should go in now.

Right *now* I'd agree with updating verbiage to refer to the 2 ways to use JS. Maybe "The primary 2 ways", or something that conveys we're only looking for 2 specific ways to use JS, while also implicitly saying "these aren't the *only* 2 ways".

**Comment by MaoShizhong** [comment]
I like that suggestion of not saying it in a way that could be interpreted as "there are 2 and only 2 ways".

**Comment by wise-king-sullyman** [comment]
Sounds good to me as well!

**Comment by crasion528** [comment]
So should I be looking at editing it along the lines of, "Identify the two ways you have learned so far to include JavaScript in a page. There is a third way but don't worry about this because it isn't best practice."?

**Comment by thatblindgeye** [comment]
@crasion528 I don't think we need to explicitly mention there's a third way. Just something like "Identify the 2 primary ways to include JavaScript"

**Comment by MaoShizhong** [comment]
An update - this work is actually no longer needed, as #26037 was a project rewrite in which the part that this issue was referencing was removed.
Sorry @crasion528, it means no work is need for this anymore, but still thank you for raising the issue in the first place!
